content/_releases/22/38.md

```diff
@@ -0,0 +1,161 @@
+into the Body field while in an Integrator mode, we changed the User Interface
+behaviour to default into JSONata mode when an `application/json` content type is
+selected for the `POST` and `PUT` HTTP calls.
+
+{% include img.html max-width="60%" url="/assets/img/RN/2238/RN2238-rest-api-v2-jsonata.png" title="REST-API-V2 step: JSONata mode for POST/PUT" %}
+
+### Run-time environment variables
+
+During the integration step execution, components exchange information with the
+platform services and database entities using already defined run-time environment
+variables. We are constantly working to standardise and add new environment variables
+to streamline the information exchange. The following environment were added to
+the run-time:
+
+*   `WEBHOOK_URI` - This environment variable will be available to each step in the flow which exchanges the data with the webhook service of the platform, eg. [WebHook](/components/webhook/) or [Flow-linking](/components/flow-linking/). This environment variable from the Tenant attributes `webhooks_domain` or from the HELM3 environment variable `WEBHOOKS_BASE_URI` if the Tenant record is missing.
+
+*   `ELASTICIO_CONTRACT_ID` - This environment variable will be available during the run-time of every execution including:
+    *   Verify credentials
+    *   Select model
+    *   Dynamic metadata
+    *   Generated samples and flow execution
+
+## Updates
+
+### Node.js Sailor version `2.7.0`
+
+A new version of Nod.JS Sailor library `2.7.0` is released. This update lays a
+preparatory groundwork towards the message persistence feature.
+
+### Java Sailor version `3.4.0`
+
+A new version of Java Sailor library `2.7.0` is released. This update lays a
+preparatory groundwork towards the message persistence feature.
+
+{% if site.data.tenant.name == "elastic.io" %}
+
+## HELM3
+
+We constantly improve different aspects of HELM3 deployments in collaboration
+with our partners. This section lists updates and improvements included in this release.
+
+### Message persistence configurations
+
+In our effort to introduce the message persistence for the integration flow steps
+we added a `.Values.global.persistentMessage` new configuration parameter to the HELM3 chart.
+
+
+### Iron-bank service Scalability
+
+The `Iron-bank` service which collects statistics from the running integration flows
+is now scalable. The service is now running as Kubernetes DeamonSet together with
+the `cadvisor`. This reduces the load on the system significantly.
+
+## OEM Related
+
+Information in this section is intended for our customers who use the
+[OEM version of the elastic.io platform](https://www.elastic.io/saas-embedded-integration/).
+
+### 
+
+
+
+
+{% endif %}
+
+## Fixed bugs
+
+*   Fixed the User Interface bug when the execution logs could not be copied.
+*   Fixed the bug in the Threads and Execution Pages when the Time picker selection would behave wrong.
```

Comment:
Fixed unexpected behavior of the Time Picker when filtering in the Threads and Execution pages.